src/middlewares/errorHttpResponseMiddleware.js

```diff
@@ -9,6 +9,7 @@ export const errorHttpResponseHandler = opts => {
     headers: {
       'Access-Control-Allow-Origin': '*',
       'Cache-Control': 'no-cache',
+      'Content-Type': 'application/json',
```

Comment:
Due to the nature of the application, mostly we are using content type as application JSON. It would be better to make it as default, and change is still doable through options parameter if needed.